Data/Sys/GameSettings/RZT.ini

```diff
@@ -1,13 +1,25 @@
 # RZTE01, RZTJ01, RZTK01, RZTP01, RZTW01 - Wii Sports Resort
 [Core]
 # Values set here will override the main Dolphin settings.
+
 [EmuState]
 # The Emulation State. 1 is worst, 5 is best, 0 is not set.
 EmulationStateId = 4
 EmulationIssues = Needs real wiimote and motion plus.
+
 [OnLoad]
 # Add memory patches to be loaded once on boot here.
+
 [OnFrame]
+
 [ActionReplay]
+
 [Video_Hacks]
 EFBEmulateFormatChanges = True
+EFBAccessEnable = True # Fix aiming
+
+[Video_Settings]
+DisableFog = True # Prevent blue box
```

Comment:
Disable fog should be removed, emulate format changes prevents the blue box in a non hacky way without the need to disable fog altogether for the game.